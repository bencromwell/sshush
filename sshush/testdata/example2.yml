---
foo_bar:
  Config:
    IdentityFile: ~/.ssh/foobar.pem
    User: ubuntu
  Hosts:
    foo_bar_aws:
      HostName: 1.2.3.4

private_foos:
  Extends: foo_bar
  Config:
    ProxyCommand: ssh foo_bar_aws nc %h %p 2> /dev/null
  Hosts:
    foobar-0: 10.25.0.1
    foobar-1: 10.25.0.2
    foobar-2: 10.25.0.3
    foobar-3: 10.25.0.4

debian_nodes:
  Extends: private_foos
  Config:
    User: admin
  Hosts:
   foobar-vpn0: 10.25.0.100
   foobar-vpn1: 10.25.0.101

direct_list:
  Hosts:
    - foo1.adm
    - bar1.adm
