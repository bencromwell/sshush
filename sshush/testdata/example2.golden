# Generated by sshush v0.0.0-dev
# From testdata/example2.yml

# foo_bar
Host foo_bar_aws
    HostName 1.2.3.4
    IdentityFile ~/.ssh/foobar.pem
    User ubuntu

# private_foos
Host foobar-0
    HostName 10.25.0.1
    IdentityFile ~/.ssh/foobar.pem
    ProxyCommand ssh foo_bar_aws nc %h %p 2> /dev/null
    User ubuntu

Host foobar-1
    HostName 10.25.0.2
    IdentityFile ~/.ssh/foobar.pem
    ProxyCommand ssh foo_bar_aws nc %h %p 2> /dev/null
    User ubuntu

Host foobar-2
    HostName 10.25.0.3
    IdentityFile ~/.ssh/foobar.pem
    ProxyCommand ssh foo_bar_aws nc %h %p 2> /dev/null
    User ubuntu

Host foobar-3
    HostName 10.25.0.4
    IdentityFile ~/.ssh/foobar.pem
    ProxyCommand ssh foo_bar_aws nc %h %p 2> /dev/null
    User ubuntu

# debian_nodes
Host foobar-vpn0
    HostName 10.25.0.100
    IdentityFile ~/.ssh/foobar.pem
    ProxyCommand ssh foo_bar_aws nc %h %p 2> /dev/null
    User admin

Host foobar-vpn1
    HostName 10.25.0.101
    IdentityFile ~/.ssh/foobar.pem
    ProxyCommand ssh foo_bar_aws nc %h %p 2> /dev/null
    User admin

# direct_list
Host bar1.adm
    HostName bar1.adm

Host foo1.adm
    HostName foo1.adm
